src/ArtGobblers.sol

```diff
@@ -482,14 +480,20 @@ contract ArtGobblers is GobblersERC721, LogisticVRGDA, Owned, ERC1155TokenReceiv
     /// @dev The price of a legendary gobbler decays as gobblers are minted. The first legendary auction begins when
     /// 1 LEGENDARY_AUCTION_INTERVAL worth of gobblers are minted, and the price decays linearly while the next interval of
     /// gobblers are minted. Every time an additional interval is minted, a new auction begins until all legendaries have been sold.
-    /// @return price of legendary gobbler, in terms of gobblers.
+    /// @dev Will revert if the auction hasn't started yet or legendaries have sold out entirely.
+    /// @return The current price of the legendary gobbler being auctioned, in terms of gobblers.
     function legendaryGobblerPrice() public view returns (uint256) {
         // Retrieve and cache various auction parameters and variables.
         uint256 startPrice = legendaryGobblerAuctionData.startPrice;
         uint256 numSold = legendaryGobblerAuctionData.numSold;
-        uint256 mintedFromGoo = numMintedFromGoo;
+
+        // If all legendary gobblers have been sold, there are none left to auction.
+        if (numSold == LEGENDARY_SUPPLY) revert NoRemainingLegendaryGobblers();
 
         unchecked {
+            // Get and cache the number of standard gobblers sold via VRGDA up until this point.
+            uint256 mintedFromGoo = numMintedFromGoo;
```

Comment:
why did you moved this to the unchecked block?

Reply:
aesthetics idk